internal/server/errors.go

```diff
@@ -48,14 +48,18 @@ func sendAPIError(c *gin.Context, err error) {
 		// log the error at info because it is not in the response
 		log = logging.L.Info()
 
+	case errors.As(err, &authnError):
+		resp.Code = http.StatusUnauthorized
+		resp.Message = authnError.Message
+
 	case errors.Is(err, data.ErrAccessKeyExpired):
 		resp.Code = http.StatusUnauthorized
 		// this means the key was once valid, so include some extra details
 		resp.Message = fmt.Sprintf("%s: %s", internal.ErrUnauthorized, err)
 
-	case errors.As(err, &authzError):
+	case errors.Is(err, access.ErrNotAuthorized):
```

Comment:
This now handles both `access.AuthorizationError` and `access.ErrNotAuthorized`, because `access.AuthorizationError` has a custom `Is` method that allows it to match `access.ErrNotAuthorized`.